src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,57 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * The Event class is a type of task.
+ *
+ * @author Chia Jeremy
+ */
+public class Event extends Task {
```

Comment:
Perhaps consider having the suffix -Task added to classes that inherit from Task for clarity? E.g EventTask